spring-integration-http/src/test/java/org/springframework/integration/http/management/IntegrationGraphControllerParserTests-context.xml

```diff
@@ -12,12 +12,6 @@
 
 	<mvc:annotation-driven />
 
-	<mvc:cors>
-		<mvc:mapping path="/foo"
-					 allowed-origins="http://foo.bar.com"
-					 allowed-methods="GET" />
-	</mvc:cors>
-
-	<graph-controller path="/foo" />
+	<graph-controller path="/foo" allowed-origins="http://foo.bar.com" />
```

Comment:
Well, now I almost remember why I have rejected similar approach before.
Change the `request.addHeader(HttpHeaders.ORIGIN, "http://foo.bar.com");` in the `testIntegrationGraphControllerParser()` to something else and test doesn't fail.

The `WebMvcConfigurerAdapter` implementation is exactly for the `@EnableWebMvc`. 
It doesn't work with XML definition. Even if it is `<mvc:annotation-driven />` we are not able to initiate the `WebMvcConfigurerAdapter` processing.
See `CorsBeanDefinitionParser` on the matter.

Investigating what we can do else...